tox.ini

```diff
@@ -53,12 +53,7 @@ commands =
 # NOTE: Any ignored errors/warnings specified below are subjective and can be changed based on
 #       common agreement of all developers contributing to this project.
 #
-# W504: line break after binary operator
-# W503: line break before binary operator (used by black formatter)
-# E203: whitespace before ':' (used by black formatter)
-#
 # TODO: move config to pyproject.toml once https://github.com/PyCQA/flake8/issues/234 is resolved
-ignore = W504,W503,E203
```

Comment:
Can we at least keep the ones that black enforces? Otherwise if someone adds a piece of code in the future that uses e.g. whitespace before a `:` then we'd just be adding these rules back.

Reply:
The key change here is using `extend-ignore` vs `ignore`, the latter overrides *all* ignored codes whereas the former simply appends to the list of things to ignore.

According to the black documentation [1], `W503` is already ignored by default by `flake8`, and black complies with `W504` I think, so `W503` is included in the ignore list and `extend-ignore` appends `E203` to it

[1] https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#id1

Reply:
Ack, so I undid my changes here except for switching "ignore" to "extend-ignore".